### ISSUE TITLE
Alarm does not work

### ISSUE THREAD

**Issue by iuhorts** [guerrerotook/securitas-direct-new-api#118]
Hello,
Since 4-5 days ago alarm stop working. Only works first alarm status change after home assistant restart.ç, after this will not work anymore untill restarting home assistant.
On integrations, securitas is showed up in orange.
Any idea?
![Screenshot_2023-02-26-18-44-21-327-edit_io homeassistant companion android](https://user-images.githubusercontent.com/54287772/221427165-ce0580fb-2d1f-4a22-ad26-21edeab45475.jpg)

Thank you!

**Comment by ToroNander** [comment]
Please add info from the log file in HA (Settings --> System --> Logs). Could have several reasons.
E.g. that u tried too often and your user is blocked. In my case it was like u show in the pic because of that. Realized through log file. Apart HA core version would be interesting.
BR

**Comment by iuhorts** [comment]
Hello!
There is the log about alarm. It says that is autoamtization but with the h.a alarm panel happens the same.

![Screenshot_2023-02-27-07-21-17-987-edit_io homeassistant companion android](https://user-images.githubusercontent.com/54287772/221489908-72abe066-2976-4430-85af-7bc8bac6dda9.jpg)
![Screenshot_2023-02-27-07-21-51-685-edit_io homeassistant companion android](https://user-images.githubusercontent.com/54287772/221489910-77f3f355-d6f9-4857-a14a-4c39377e5919.jpg)

**Comment by guerrerotook** [comment]
Did you do the 2FA and received an SMS to login into the integration?

**Comment by iuhorts** [comment]
Nope, did I?


> Did you do the 2FA and received an SMS to login into the integration?

**Comment by iuhorts** [comment]
> Did you do the 2FA and received an SMS to login into the integration?

Yes had it done. With furter investigations i found the problem is when i try to reload the integration i do this after each activation/deactivation because most of times it fails and doesan't refresh until next chack(20minutes).
I use: Home Assistant Core Integration: Reload config entry after this i got an error.
![Screenshot_2023-03-19-09-16-26-922_io homeassistant companion android](https://user-images.githubusercontent.com/54287772/226162492-19c2dbab-8068-4c81-bdb0-1ef982ff3e37.jpg)

**Comment by Borty97** [comment]
Did you wrote your code like this in `configuration.yaml`?:
`securitas:
username: !secret securitas_direct_username
password: !secret securitas_direct_password
code: !secret securitas_direct_code
country: ES`
And in your `secrets.yaml`:
`securitas_direct_username: username
securitas_direct_password: password
securitas_direct_code: "code"`

Remember to write the code with colons

**Comment by iuhorts** [comment]
> Did you wrote your code like this in `configuration.yaml`?: `securitas: username: !secret securitas_direct_username password: !secret securitas_direct_password code: !secret securitas_direct_code country: ES` And in your `secrets.yaml`: `securitas_direct_username: username securitas_direct_password: password securitas_direct_code: "code"`
> 
> Remember to write the code with colons

Mm, thank you, the thing is i've it configurated trough integration wizard, that you fill the textbox.
Need to set it into configuration.yaml?

**Comment by iuhorts** [comment]
> Yes, you need to define this variables first in `configuration.yaml` and then restart Home Assistant. Once done, it will discover the integration and it will automatically show your phone to send the 2FA message.

Yes, but you can do it in wizard, and if you do in wizard not need in configuration.yaml
